python-sdk/example_dags/example_datasets.py

```diff
@@ -34,7 +34,7 @@
 # [END dataset_table]
 
 top_animations_table = Table(name="top_animation", conn_id="sqlite_default")
-START_DATE = datetime(2022, 9, 1)
+START_DATE = datetime(2016, 9, 1)
```

Comment:
Why was this needed? The date was already in past, Sep 9, 2022?

Reply:
Nope, START_DATE was used to initialize dag.

![Screenshot 2022-11-01 at 7 28 54 PM](https://user-images.githubusercontent.com/13021213/199250968-c90ec2f9-92e9-4b5d-9c35-504d004ace19.png)

Reply:
ahh yes, but weird that fixed the issue in CI. Maybe an intermittent issue